packages/interactivity/src/index.js

```diff
@@ -7,8 +7,17 @@ import { init } from './router';
 export { store } from './store';
 export { directive, getContext, getElement } from './hooks';
 export { navigate, prefetch } from './router';
-export { h as createElement } from 'preact';
-export { useEffect, useContext, useMemo } from 'preact/hooks';
+export {
+	useWatch,
+	useInit,
+	useEffect,
+	useLayoutEffect,
+	useCallback,
+	useMemo,
```

Comment:
Let's reexport `useState` and `useRef`, which now can be used inside `data-wp-run`.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/57805/commits/56566cf896713b9f34b4fa6d6ead48749a8477af